src/ignitus-Shared/ignitus-DesignSystem/ignitus-Layout/ignitus-NotFound/Components/index.tsx

```diff
@@ -20,9 +20,11 @@ export const Notfound = () => (
             </span>
           </S.StyledParagraph>
         </S.Message>
-        <S.StyledButton size="large" category="primary">
-          <Link to="/"> Home Page </Link>
-        </S.StyledButton>
+        <BrowserRouter>
```

Comment:
Why you used that?

Reply:
Because the tests were failing otherwise. The error it showed was:
`Invariant failed: You should not use <Link> outside a <Router>`
That's why, I encapsulated the <Link> inside <BrowserRouter> for a workaround.

Reply:
@Dishebh then you need to import Router in your test, not in your component.
Something like this.

```
describe('<Notfound />', () => {
  it('should render', () => {
    const wrapper = shallow(
      <Router>
        <Notfound />
      </Router>,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
```

cc @Marvin9

Reply:
Thanks! Will keep that in mind.